frontend/src/features/directory/components/FilterableVendorTable.tsx

```diff
@@ -48,12 +48,15 @@ function FilterableVendorTableContent({ uniqueRegions, tags, vendors, favoriteVe
     return vendors.filter((vendor) => {
       const matchesRegion = selectedRegion ? vendor.metro_region === selectedRegion : true;
       const matchesTravel = travelsWorldwide ? vendor.travels_world_wide : true;
-      const matchesSkill = selectedSkill ? vendor.tags
-        .some((tag) => tag.display_name?.toLowerCase() === selectedSkill.toLowerCase())
+      const matchesSkills = selectedSkills ? !selectedSkills
```

Comment:
What do you think about treating multiselect as OR instead of AND? That's more intuitive IMO, and also what zola does https://www.zola.com/wedding-vendors/search/phoenix-az--wedding-hair-makeup--hair--henna-art--makeup